vizro-core/docs/pages/user_guides/tabs.md

```diff
@@ -1,22 +1,30 @@
 # How to use tabs
 
-Tabs organize and separate groups of related content in a dashboard, letting users switch between different sections or views.
-They are essentially a way of stacking multiple [`Containers`](container.md) on top of each other, and letting the user switch between them.
-[`Containers`][vizro.models.Container] enable the grouping of page components into sections and subsections.
+[`Tabs`][vizro.models.Tabs]  organize and separate groups of related content in a dashboard, letting users switch between different sections or views.
+They are essentially a way of putting multiple [`Containers`][vizro.models.Container] in the same screen space, and letting the user switch between them.
+`Containers` enable the grouping of page components into sections and subsections, see this user guide [here](container.md) for more information.
```

Comment:
```suggestion
`Containers` enable the grouping of page components into sections and subsections; see our [user guide on `Containers`](container.md) for more information.
```

Reply:
See e.g. https://github.com/kedro-org/kedro/wiki/Kedro-documentation-style-guide#links 🙂

Reply:
Ah nice, I didn't even know that wiki existed. Super helpful! 🚀

Reply:
I've fixed it in other places as well where we have this pattern of "see user guide on.."